docs/src/Run/AccountService/RunAccountService.md

```diff
@@ -0,0 +1,49 @@
+<!-- TODO: Update for Builder Track, see: https://github.com/ProjectLibertyLabs/gateway/issues/630 -->
+# Running Account Service
+
+## Prerequisites
+
+In order to run this project you need:
+
+- [Node.js](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
+
+## Environment Variables
+
+Modify any environment variables in the `.env` file as needed. The complete set of environment variables is documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](../../env-files/account.template.env).
+
```

Comment:
Note: these links don't work for me.  Probably a permissions thing.

Reply:
I'll double check them, but they will not be valid until the changes are published.